plugins/thumbnail/lib/model/thumbnailFactory/kBaseResizeAdapter.php

```diff
@@ -136,6 +134,20 @@ protected function checkIfOldApiCachedExists($contentPath, $thumbDirs)
 			$currPath = $contentPath . myContentStorage::getGeneralEntityPath(self::ENTITY_NAME_PREFIX . $thumbDir, $entry->getIntId(), $this->thumbName, $this->entryThumbFilename , $version );
 			if (file_exists($currPath) && @filesize($currPath))
 			{
+				if($currPath != $this->finalThumbPath)
+				{
+					$moveFileSuccess = kFile::moveFile($currPath, $this->finalThumbPath);
+					if($moveFileSuccess)
+					{
+						return array (true, $this->finalThumbPath);
+					}
+					else
+					{
+						KalturaLog::Error("Failed to move thumbnail from [$currPath] to [$this->finalThumbPath], will return oldPath");
```

Comment:
In this case - 
1. Error is not requires as we do not really need the traceback nor the parameters 
2. Is it correct to return value rather then flase?